scripts/wait-for-discovery.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
what is this script for?  is it used for the docker-compose environment?

Reply:
Yes.  In testing the HSM layer I need discovery to be complete before running the tests.   I was originally doing this in the HSM layer test code, but would guess over time that other tests will have the same requirement.  Thus, this script causes the test to wait until HSM discovery is done before running any of the tests.